CONTRIBUTORS.md

```diff
@@ -1,59 +1,61 @@
-33. [Ram Kumar](#ram-kumar)
-34. [Franklin Ohaegbulam](#franklin-ohaegbulam)
-35. [Yugal Agarwal](#yugal-agarwal)
-36. [Hardik Jain](#hardik-jain)
-37. [Caleb Okechukwu](#caleb-okechukwu)
-38. [Tejesh Annavarapu](#tejesh-annavarapu)
-39. [Hadar Hubara](#hadar-hubara)
-40. [Alexandros Yiangou](#alexandros-yiangou)
-41. [Tarun Singh](#tarun-singh)
-42. [Kayode Oluwasegin](#kayode-oluwasegun)
-43. [Daniel Udechukwu](#daniel-udechukwu)
-44. [Sriniketh J](#sriniketh-j)
-45. [Dheeraj Ghosh](#dheeraj-ghosh)
-46. [Ijeoma Onwuka](#ijeoma-onwuka)
-47. [Jerry Zhu](#jerry-zhu)
-48. [Olumide Nwosu](#olumide-nwosu)
-49. [Aryan Shaw](#aryan-shaw)
-50. [Abhilipsa Sahoo](#abhilipsa-sahoo)
-51. [Jamilu Jibrilla](#jamilu-jibrilla)
-52. [Awosise Oluwaseun](#awosise-oluwaseun)
-53. [Issac Jacob](#issac-jacob)
-54. [Adeola Adetunji](#adeola-adetunji)
+- [Contributors](#contributors)
+  - [Joshua Poddoku](#joshua-poddoku)
+  - [Abel Mathew](#abel-mathew)
+  - [Loveth Omokaro](#loveth-omokaro)
+  - [Rakesh Potnuru](#rakesh-potnuru)
+  - [Siddharth Shivkumar](#siddharth-shivkumar)
+  - [Gurjeet Singh Virdee](#gurjeet-singh-virdee)
+  - [Pranai Raj](#pranai-raj)
+  - [Sankalp Swami](#sankalp-swami)
+  - [Palak Jain](#palak-jain)
+  - [Asmit Malakannawar](#asmit-malakannawar)
+  - [Aniruddha Inge](#aniruddha-inge)
+  - [Garima Mangal](#garima-mangal)
+  - [Snehangsu De](#snehangsu-de)
+  - [Fidal Mathew](#fidal-mathew)
+  - [Anjani Kumar Panchal](#anjani-kumar-panchal)
+  - [Varghese Jose](#varghese-jose)
+  - [Musab Habeeb](#musab-habeeb)
+  - [Shubham Vidhawan](#shubham-vidhawan)
+  - [Danish Saleem](#danish-saleem)
+  - [Joseph Jose](#joseph-jose)
+  - [Segun Ajibola](#segun-ajibola)
+  - [Victor Eke](#victor-eke)
+  - [Benjamin Thorpe](#benjamin-thorpe)
+  - [Timilehin Omolana](#timilehin-omolana)
+  - [Andrew Osei Owusu](#andrew-osei-owusu)
+  - [Victor He](#victor-he)
+  - [Bhumi Khokhani](#bhumi-khokhani)
+  - [Akande Olalekan Toheeb](#akande-olalekan-toheeb)
+  - [Collins Rono](#collins-rono)
+  - [Tracy Nuwagaba](#tracy-nuwagaba)
+  - [M A Alim](#m-a-alim)
+  - [Ganesh Patil](#ganesh-patil)
+  - [Ram Kumar](#ram-kumar)
+  - [Franklin Ohaegbulam](#franklin-ohaegbulam)
+  - [Yugal Agarwal](#yugal-agarwal)
+  - [Hardik Jain](#hardik-jain)
+  - [Caleb Okechukwu](#caleb-okechukwu)
+  - [Tejesh Annavarapu](#tejesh-annavarapu)
+  - [Hadar Hubara](#hadar-hubara)
+  - [Alexandros Yiangou](#alexandros-yiangou)
+  - [Tarun Singh](#tarun-singh)
+  - [Kayode Oluwasegun](#kayode-oluwasegun)
+  - [Daniel Udechukwu](#daniel-udechukwu)
+  - [Sriniketh J](#sriniketh-j)
+  - [Dheeraj Ghosh](#dheeraj-ghosh)
+  - [Ijeoma Onwuka](#ijeoma-onwuka)
+  - [Jerry Zhu](#jerry-zhu)
+  - [Olumide Nwosu](#olumide-nwosu)
+  - [Aryan Shaw](#aryan-shaw)
+  - [Abhilipsa Sahoo](#abhilipsa-sahoo)
+  - [Jamilu Jibrilla](#jamilu-jibrilla)
+  - [Awosise Oluwaseun](#awosise-oluwaseun)
+  - [Issac Jacob](#issac-jacob)
+  - [Adeola Adetunji](#adeola-adetunji)
+  - [Ezinne Anne Emilia](#ezinne-anne-emilia)
```

Comment:
Can you sync it with the recent changes and only add your name(+ description) and resend the PR?

Reply:
sorry, am confused by what you mean by sync changes. how do I do that?